### ISSUE TITLE
Formatting on persona section

### ISSUE THREAD

**Issue by KateMcCall-Kiley** [presidential-innovation-fellows/open-opportunity-data#16]
Would love to tighten up formatting in this section. Perhaps making each persona smaller and so that there is a grid of 6 equal sections, 3 across, 2 down. ?

**Comment by rossdakin** [comment]
It's currently this:

<table>
  <tr>
    <td rowspan="2">Overview</td>
    <td>Quotes</td>
    <td>Actions</td>
  </tr>
  <tr>
    <td>Needs</td>
    <td>Goals</td>
  </tr>
</table>

The "rows" aren't equal because it's a little difficult to make that happen and make it look good in a mobile view (see how the sections stack in a narrow web view?).

**Comment by KateMcCall-Kiley** [comment]
Ah I see - the mobile looks great yea.

Is there a way to break up the overview into: 
Icon and Name
Description
?
(This can be for later as well)

**Comment by rossdakin** [comment]
Yeah, we can do that if there's another section you'd like to add.
